language-design.md

```diff
@@ -49,6 +119,12 @@ In what way is an API a language?
 
 **Response**
 
+An API has many language characteristics talked about in the articles.  It is
+growable in the way Steele states that langauges should be and often one can use
+an API to build more APIs.  Moreover, often communities develop APIs together
+and certain communities like certain APIs and not others in the same way certain
+communities like particular languages but not others in the way talked about in
+Yang and Rabkin's article.
 
```

Comment:
The last point is interesting, hadn't thought of that. Do you think APIs are susceptible to the themes explored in Yang and Rabkin's article as well?